src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  padding-left: 50px;
+  padding-top: 10px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-around;
+  gap: 20px;
+  align-items: baseline;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  color: #000;
+  line-height: 14px;
+}
+
+.nav__link {
+  position: relative;
+  padding: 22px 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__item:last-child {
+  padding-right: 22px;
```

Comment:
indents should be the same
![image](https://user-images.githubusercontent.com/64437769/142622864-7893d813-38fb-48ee-b38b-2d6aa38e56f8.png)